test/server/lib/tags.test.ts

```diff
@@ -0,0 +1,19 @@
+import { assert } from 'chai';
+
+import { sanitizeTags } from '../../../server/lib/tags';
+
+describe('server/lib/tags', () => {
+  it('sanitize tags', () => {
+    const tags = [
+      'OpenCollective',
+      '#OpenCollective',
+      'o#penCollective',
+      'OpenCollective#',
+      '##OpenCollective',
+      ' test ',
+      'double  space',
```

Comment:
Some interesting cases to add

```suggestion
      '    ', // Empty should be removed
      'OpEn-SoUrCe, // Should be unified under "open source"
```

Reply:
Added these!